requirements.txt

```diff
@@ -1,5 +1,5 @@
 
-streamlit!=1.19.0,!=1.18.1,!=1.18.1rc1,!=1.18.0
+streamlit>=1.20.0
```

Comment:
This will break the apps that are using an older version of Streamlit.

I'm not sure what's the best strategy here. Probably it is better to break them and fix them.